src/main/webapp/javascript/add-event.js

```diff
@@ -0,0 +1,145 @@
+var data = {};
+var id = getEventIdByUrl("id");
+
+function goToTheMainPage() {
+    location.href = '/app/start.jsp';
+}
+
+function getDataFromTextarea() {
+    var name = document.getElementById("name");
+    data["name"] = (name.value).trim();
+    var description = document.getElementById("description");
+    data["description"] = (description.value).trim();
+    var agenda = document.getElementById("agenda");
+    data["agenda"] = (agenda.value).trim();
+    var eventdate = document.getElementById("datepicker");
+    data["date"] = eventdate.value;
+    var eventtime = document.getElementById("timepicker");
+    data["time"] = (eventtime.value).trim();
+
+    var statusClosed = document.getElementById("closed");
+    var statusFinished = document.getElementById("finished");
+    var statusOpen = document.getElementById("open");
+    if (statusClosed.checked) {
+        data["status"] = "CLOSED"
+    }
+    if (statusFinished.checked) {
+        data["status"] = "FINISHED"
+    }
+    if (statusOpen.checked) {
+        data["status"] = "OPEN"
+    }
+
+    if (id) {
+        data["id"] = id;
+    }
+    return data;
+}
+
+function checkNonEmptyInput(data) {
+    return validateField(data.name, "Name")
+        && validateField(data.description, "Description")
+        && validateField(data.agenda, "Agenda")
+        && validateField(data.date, "Date")
+        && validateField(data.time, "Time");
+
+}
+
+function validateField(field, message) {
+    if(field === "") {
+        alert("Please input Event " + message);
+        return false;
+    }
+    return true;
+}
+
+function switchPageStatus() {
+    if (id) {
+        getEventData();
+        document.getElementById("edit").classList.remove("w3-hide");
+        document.getElementById("update").classList.remove("w3-hide");
+        document.getElementById("save").classList.add("w3-hide");
+        document.getElementById("update").classList.remove("w3-hide");
+        document.getElementById("setStatus").classList.remove("w3-hide");
+        document.getElementById("statuses").classList.remove("w3-hide");
+        document.getElementById("save").classList.add("w3-hide");
+        document.getElementById("add").classList.add("w3-hide");
+    }
+    else {
```

Comment:
`else {` should be placed on the same line as the preceding bracket: `} else {`.